requirements.txt

```diff
@@ -4,6 +4,7 @@ chardet==3.0.4
 Click==7.0
 Flask==1.1.1
 get==2019.4.13
+gunicorn==19.9.0
```

Comment:
This is what Heroku suggests using, but for the moment, we'll just try like normal.